public/components/marketing-stats/2.html

```diff
@@ -0,0 +1,36 @@
+<section role="region" aria-label="Statistics" class="px-4 py-16 mx-auto">
+    
+    <div class="text-center md:w-1/2 lg:w-1/2 w-full mx-auto">
+      <h1 class="font-extrabold text-3xl"> Most Prominent In e-commerce </h1>
+      <p class="mt-4 text-gray-500 w-fit mx-auto"> Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. 
+      visual form of a document or a typeface without relying on meaningful content.
+      </p>
+    </div>
+
+    <div class="grid grid-cols-2 row-gap-8 gap-y-8 md:grid-cols-4 mt-10">
+      <div class="text-center md:border-r">
```

Comment:
Could use `md:divide-x` on the parent element, instead of `md:border-r` on child elements.